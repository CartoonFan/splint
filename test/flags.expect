
flags.c:1:9: Variable x initialized to type char, expects int: 'c'
flags.c:5:9: Variable y
    initialized to type
    char, expects int:
    'c'
flags.c:7:18: Warning:
    Value for linelen
    must be at least 20
    (given 13)
flags.c:11:5:
    Internal identifier
    xx2 is not
    distinguishable
    from xx1 in the
    first 2 characters
    (xx)
   flags.c:10:5:
   Declaration of
   xx1
flags.c:13:18:
    Stylized comment
    attempts to restore
    flag linelen.  A
    flag for setting a
    value cannot be
    restored.
flags.c:15:17: Stylized comment attempts to set global flag dump.  A
                  global flag cannot be set locally.
flags.c:16:22: Flag macrovarprefix (in stylized comment) must be
                  followed by an argument
flags.c:17:26: Flag macrovarprefix (in stylized comment) must be
                  followed by an argument
flags.c:18:20: Flag linelen must be followed by a positive number
                  number.  Followed by asdf

Finished LCLint checking --- 8 code errors found, as expected

flags.c:1:9: Variable x initialized to type char, expects int: 'c'
flags.c:5:9: Variable y initialized to type char, expects int: 'c'

Finished LCLint checking --- 2 code errors found, as expected
