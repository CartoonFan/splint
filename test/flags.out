
Cannot find standard library: ansi.lcd
     Check LARCH_PATH environment variable.
flags.c:1:9: Variable x initialized to type char, expects int: 'c'
flags.c:5:9: Variable y
    initialized to type
    char, expects int:
    'c'
flags.c:7:18: Warning:
    Value for linelen
    must be at least 20
    (given 13)
flags.c:11:5:
    Internal identifier
    xx2 is not
    distinguishable
    from xx1 in the
    first 2 characters
    (xx)
   flags.c:10:5:
   Declaration of
   xx1
flags.c:13:18:
    Semantic comment
    attempts to restore
    flag linelen.  A
    flag for setting a
    value cannot be
    restored.
flags.c:15:17: Semantic comment attempts to set global flag dump.  A
                  global flag cannot be set locally.
flags.c:16:22: Flag macrovarprefix (in semantic comment) must be
                  followed by an argument
flags.c:17:26: Flag macrovarprefix (in semantic comment) must be
                  followed by an argument
flags.c:18:20: Flag linelen must be followed by a positive number
                  number.  Followed by asdf

Finished LCLint checking --- 8 code errors found, as expected

Cannot find standard library: ansi.lcd
     Check LARCH_PATH environment variable.
flags.c:1:9: Variable x initialized to type char, expects int: 'c'
flags.c:5:9: Variable y initialized to type char, expects int: 'c'

Finished LCLint checking --- 2 code errors found, as expected
